misc/dbt-materialize/dbt/include/materialize/macros/adapters.sql

```diff
@@ -100,6 +100,15 @@
     {%- endif %};
 {%- endmacro %}
 
+{% macro materialize__alter_relation_comment(relation, comment) -%}
+  {% set escaped_comment = postgres_escape_comment(comment) %}
+  {% if relation.type == 'materializedview' -%}
```

Comment:
Before seeing your PR, I'd gone a bit fancier locally and added a property to `MaterializeRelation` in `relation.py`:

```python
@property
    def is_materialized_view(self) -> bool:
        return self.type == MaterializeRelationType.MaterializedView
```

then used:

```sql
  {%- if relation.is_materialized_view -%}
```

It feels mostly unnecessary as is, but might be more future-proof since we'll need to support both `materialized_view` and `materializedview` as synonyms for the materialization type.

Reply:
Introducing this change in [here](https://github.com/MaterializeInc/materialize/pull/21889/files#diff-be58835b29f372ae6b8afecd5ba5ae01315af4a73524587246e62283eb784aa1) for the v1.6.0 upgrade.